charts/opentelemetry-operator/templates/deployment.yaml

```diff
@@ -15,6 +15,7 @@ spec:
   template:
     metadata:
       annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
Does this annotation always need to be preset or is this scenario specific to the way you're managing certificates?  I'm trying to understand if this annotations is needed for everybody or if it could be handled case-by-case by updating the `podAnnotations` configuration.

Reply:
oh, your'e right, make `rollme: {{ randAlphaNum 5 | quote }}` as configurable by `podAnnotations`  👍

Reply:
I update according to your idea, but  we still need to ignore `roleName ` during `make check-examples` stage.